skaffold.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: skaffold/v4beta12
+kind: Config
+metadata:
+  name: cern-vre
+deploy:
+  helm:
+    releases:
+      - name: cern-vre
```

Comment:
maybe "escape-vre" is better, or even "my-escape-vre" as a default release name.  Whatever this thing is, it won't actually be the CERN VRE :)

Reply:
I see, interesting question. 

For me, "VRE" is a generic term which I see throughout the related domains. So I assumed that name of the software product is "CERN VRE". 

It also coincides with the name of the CERN instance.

However, we have now ESCAPE VRE, CERN VRE, ETAP VRE, and I wonder.

@garciagenrique @Soap2G what do you think? Should "CERN" be anywhere in the name of the generic VRE chart?

Reply:
Renamed to ESCAPE VRE.